openedx/core/djangoapps/course_groups/views.py

```diff
@@ -9,19 +9,31 @@
 from django.contrib.auth.models import User
 from django.core.exceptions import ValidationError
 from django.core.paginator import EmptyPage, Paginator
+from django.db import transaction
 from django.urls import reverse
 from django.http import Http404, HttpResponseBadRequest
+from django.utils.decorators import method_decorator
 from django.utils.translation import ugettext
 from django.views.decorators.csrf import ensure_csrf_cookie
 from django.views.decorators.http import require_http_methods, require_POST
+from edx_rest_framework_extensions.authentication import JwtAuthentication
 from opaque_keys.edx.keys import CourseKey
+from rest_framework import status, permissions
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework_oauth.authentication import OAuth2Authentication
 from six import text_type
 
 from courseware.courses import get_course_with_access
 from edxmako.shortcuts import render_to_response
+from util.file import FileValidationException, store_uploaded_file, course_and_time_based_filename_generator
 from util.json_request import JsonResponse, expect_json
 
-from . import cohorts
+from instructor_task.api_helper import submit_task
+from instructor_task.tasks import cohort_students
```

Comment:
Import these as ``from lms.djangoapp.instructor_task.api_helper import submit_task`` and ``from lms.djangoapp.instructor_task.tasks import cohort_students``.

Reply:
I usually issues with the `lms.djangoapps` prefix. Anyway, nothing changes when modifying this, that's really odd.

Reply:
I think the import paths just need to be consistent. When I've faced similar issues that's usually been the problem.